video/out/w32_common.c

```diff
@@ -1315,6 +1315,29 @@ static void update_cursor_passthrough(const struct vo_w32_state *w32)
     }
 }
 
+static void update_native_touch(const struct vo_w32_state *w32)
+{
+    if (w32->parent)
+        return;
+
+    // Block unwanted gestures (press and hold etc)
+    GESTURECONFIG gc = {0};
+    if (w32->opts->native_touch) {
+        RegisterTouchWindow(w32->window, 0);
+        gc.dwBlock = GC_ALLGESTURES;
+    } else {
+        UnregisterTouchWindow(w32->window);
+        gc.dwWant = GC_ALLGESTURES;
+    }
+    SetGestureConfig(w32->window, 0, 1, &gc, sizeof(gc));
+}
```

Comment:
thought: this should trigger `MP_TOUCH_RELEASE_ALL` when touch is disabled.
or maybe that should be up to the core